src/components/header/renderer.tsx

```diff
@@ -129,83 +109,122 @@ class Header extends React.Component<Props, State> {
         {'New'}
       </div>
     );
-    const vega = (
-      <div className='vega'>
-        {Object.keys(VEGA_SPECS).map((specType, i) => {
-          const specs = VEGA_SPECS[specType];
-          return (
-            <div className='itemGroup' key={i}>
-              <div className='specType'>{specType}</div>
-              <div className='items'>
-                {specs.map((spec, j) => {
-                  return (
-                    <div key={j} onClick={() => this.onSelectVega(spec.name)} className='item'>
-                      <div style={{backgroundImage: `url(images/examples/vg/${spec.name}.vg.png)`}} className='img'/>
-                      <div className='name'>{formatExampleName(spec.name)}</div>
-                    </div>
-                  );
-                })}
+    const vega = (closePortal) => {
+      return (
+        <div className='vega'>
+          {Object.keys(VEGA_SPECS).map((specType, i) => {
+            const specs = VEGA_SPECS[specType];
+            return (
+              <div className='itemGroup' key={i}>
+                <div className='specType'>{specType}</div>
+                <div className='items'>
+                  {specs.map((spec, j) => {
+                    return (
```

Comment:
Please use the more compressed markup form the left. It's easier to read.

Reply:
What do you mean by compressed markup?

Reply:
Yeah, I like it more as it was before.

Reply:
Thanks for changing it!

Reply:
Fixed.

Reply:
I've fixed the markdowns as you wanted.